### ISSUE TITLE
Crash when reloading movies Library

### ISSUE THREAD

**Issue by mpwg** [Komet/MediaElch#1316]
**Describe the bug**
Every time I reload the Movies library, mediaelch crashes

**To Reproduce**
1. Go to ‚Movies.'
2. Click on ‚Reload.‘
3. See Crash

**MediaElch Version:**
 - [ ] 2.8.11-dev (nightly)
 - [x] 2.8.10 stable

**Operating System:**
 - [ ] Windows
 - [x] macOS
 - [ ] Linux (distribution: _._)

**Additional context**
I’ve added [macOS Crash Report.txt](https://github.com/Komet/MediaElch/files/6433990/macOS.Crash.Report.txt)

**Comment by mpwg** [comment]
i had an empty Folder that caused the crash (forgot to add the file)

When I add the file, it works fine

**Comment by bugwelle** [comment]
Thank you for reporting this issue. May be related to #1315 

Could you tell me what the folder structure was? (including the empty folder)? :)

**Comment by mpwg** [comment]
The relevant folder was

`/Volumes/Mediathek/Theater/Weekend im Paradies (1981)`
(its on a file share mounted via AFP)

There are several sub-folders in `/Volumes/Mediathek` (Like `Filme`, `TV-Serien`, …) and several other Folders inside `/Volumes/Mediathek/Theater/`

**Comment by bugwelle** [comment]
Thanks. Can't reproduce this, yet (also on macOS; macOS 11.2.3).

Could you tell me:

1. What folders have you configured in MediaElch (type/sep. folders)?
2. How many movies do you have?
3. Did you simply add a file to `/Volumes/Mediathek/Theater/Weekend im Paradies (1981)` to fix this issue?

Regards,
Andre. :-)

**Comment by mpwg** [comment]
1. you mean this info? 
<img width="902" alt="1" src="https://user-images.githubusercontent.com/68284225/117296539-76a71480-ae75-11eb-82be-85072aeaa22c.png"> 
<img width="902" alt="2" src="https://user-images.githubusercontent.com/68284225/117296567-80c91300-ae75-11eb-9793-a70c8c990cbc.png">

2. There are 1155 movies in the library 
3. Yep, I added the actual movie ;-) (forgot it)

**Comment by bugwelle** [comment]
Thanks! I'll prepare a Nightly version. I have a suspicion where the issue is.

**Comment by bugwelle** [comment]
Oh and, can you reproduce the bug if you remove the movie from `Weekend im Paradies (1981)` again?

**Comment by bugwelle** [comment]
_If (and only if)_ you can reproduce the issue, please try this Nightly version:

https://mediaelch-downloads.ameyering.de/snapshots/macOS/MediaElch_macOS_2.8.11-dev_2021-05-06_12-36_git-fix_movie-reload-crash-6b9b2370.dmg

It has a small change. It may or may not fix your issue. But to confirm whether it fixes the issue, it would be great if you can first reproduce the issue.

Thank you. :-)

**Comment by bugwelle** [comment]
Reproduced the issue. Could it be that you do not have any TV shows in MediaElch loaded? :-)

**Comment by Serojin** [comment]
Sorry if it's poor form, but I wanted to +1 this. I too have this issue (similar OS and circumstance). Some information that might help: I kept my movies in one big folder. I used the organize function of MediaElch to drop them in their own folders which seemed to avert the bug when loading movies list. Unfortunately if you scrape that new list, it'll crash (but make progress - I had to re-open and scan a dozen times to get them all in there)
